### PR TITLE
feat(dart): improve keybindings for better Flutter control

### PR DESCRIPTION
**Pull request by victorblq**
Improving the keybinds for Flutter control on a dart file buffer.

------
- [x] I've searched the issue tracker to ensure this was not PRed before.
- [x] My PR is not on [the do-not-PR list](https://doomemacs.org/d/do-not-pr) for this project.
- [x] My commits conform to [the git conventions](https://doomemacs.org/d/git-conventions).

### CODE REVIEW

**Review by ericdallo — COMMENTED** [review]
LGTM

File: modules/lang/dart/config.el [hunk 1]
```diff
@@ -32,8 +32,16 @@
...
+        (:localleader
+         (:prefix ("f" . "Flutter")
+          :desc "Flutter run"
+          "f" #'flutter-run
+          :desc "Flutter quit"
+          "q" #'flutter-quit
+          :desc "Flutter hot reload"
+          "r" #'flutter-hot-reload
+          :desc "Flutter hot restart"
+          "R" #'flutter-hot-restart))))
```
Comment (hlissner) [inline modules/lang/dart/config.el:44]: It's a mote busier than it needs to be, and is inconsistent with other uses of `map!` across Doom's codebase. Could you inline these and remove a level of parentheses?

I.e.
```lisp
  (map! :after dart-mode
        :map dart-mode-map
        :localleader
        (:prefix ("f" . "Flutter")
         :desc "Flutter run"         "f" #'flutter-run
         :desc "Flutter quit"        "q" #'flutter-quit
         :desc "Flutter hot reload"  "r" #'flutter-hot-reload
         :desc "Flutter hot restart" "R" #'flutter-hot-restart)))
```

(Side note: I'm not sure we _really_ need these descriptions, considering they (practically) match their command's names, but I'll revisit that when I get around to rethinking how Doom uses which-key)

**Review by hlissner — CHANGES_REQUESTED** [review]

File: modules/lang/dart/config.el (hunk 1 above)
Comment (ericdallo) [reply modules/lang/dart/config.el:44]: Agreed, the command names are pretty clear and easier if user wants to debug a command

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Thanks for your help!

**Pull request merged**